src/CoreTelephony/CTEnums.cs

```diff
@@ -11,7 +11,7 @@ public enum CTErrorDomain {
 		Mach = 2,
 	}
 
-	[Obsoleted (PlatformName.iOS, 14,0, message: Constants.UseCallKitInstead)]
```

Comment:
Did iOS not really obsolete this? I'm a bit confused by this.

Reply:
It's not: https://developer.apple.com/documentation/coretelephony/ctcellulardatarestrictedstate?language=objc

Some types say that you should use CallKit instead, but it looks like that's only limited to certain types: https://developer.apple.com/documentation/coretelephony/ctcallcenter?language=objc